index.html

```diff
@@ -171,8 +177,9 @@
       other features of the Thing Description that go beyond the constraints
       of the HTTP Baseline Profile, however the interoperability guarantees of the HTTP Baseline Profile
       hold only for the <a>HTTP Baseline Profile</a> subset.
-
-
+    </p>
+    <p>Future versions of this document may contain other profiles,
+      e.g. a profile for digital twins and a profile for resource constrained devices.</p> 
```

Comment:
I would prefer not to list digital twins as an example because I'm not yet convinced that's a good example use case for a profile. Better to just use the constrained devices example which is less contentious.

Reply:
Will change to "profiles for other protocols and profiles for resource constrained devices."

Reply:
Profile call on Wed 7h: remove lines 180-182